app/api/leaderboard/functions.ts

```diff
@@ -48,14 +48,15 @@ export const getLeaderboardData = async (
     .filter((contributor) => contributor.summary.points)
     .filter((contributor) => {
       if (role.length === 0) return true;
-      if (role.includes("core") && contributor.core) return true;
-      if (role.includes("intern") && contributor.intern) return true;
-      if (role.includes("operations") && contributor.operations) return true;
+      if (role.includes("core") && contributor.role === "core") return true;
+      if (role.includes("intern") && contributor.role === "intern") return true;
+      if (role.includes("operations") && contributor.role === "operations")
+        return true;
       if (
         role.includes("contributor") &&
-        !contributor.core &&
-        !contributor.intern &&
-        !contributor.operations
+        contributor.role !== "core" &&
+        contributor.role !== "intern" &&
+        contributor.role !== "operations"
       )
         return true;
       return false;
```

Comment:
This could now be replaced with a single line right?

```jsx
return role.length === 0 || role.includes(contributor.role)
```

Reply:
yupp